include/xten/Transform/Passes.td

```diff
@@ -50,4 +50,20 @@ def XTenMinimizeLiveTensors : Pass<"xten-minimize-live", "func::FuncOp"> {
   ];
 }
 
+def XTenCanonicalizePass : Pass<"xten-canonicalize"> {
+  let summary = "Perform XTen specific canonicalizations on the IR";
+  let description = [{
+    Pass that performs XTen specific foldings & canonicalizations to bring the
```

Comment:
```suggestion
    Pass that performs XTenNN specific foldings & canonicalizations to bring the
```
I know its confusing but we have a dialect named XTen and another named XTenNN with our QDQ ops defined.